### ISSUE TITLE
[FVT]:sync up rinv manpage and help information

### ISSUE THREAD

**Issue by tingtli** [xcat2/xcat-core#3276]
With the latest xCAT 2.13.5 build. I found rinv manpage and help information is not the same. See the BMC/MPA specific part.

The manpage has more options than help information.

```
[root@c910f02c01p13 ~]# man rinv
   BMC/MPA specific:
       rinv noderange {pci|model|serial|asset|vpd|mprom|deviceid|guid|firm|diag|dimm|bios|mparom|mac|all}
```

```
[root@c910f02c01p13 ~]# rinv -h
Usage:
    BMC specific:
       rinv <noderange> [mprom|deviceid|uuid|guid|vpd|dimm|all]
```

**Comment by tingtli** [comment]
@zet809  Please take a look, thanks!

**Comment by whowutwut** [comment]
@gurevichmark If you have some down time here and there to look into this, please feel free to take the issue.

**Comment by zet809** [comment]
The manpage is more accurate, but seems pci/bios/mac are dropped.
```
BMC/MPA specific:
       rinv noderange {pci|model|serial|asset|vpd|mprom|deviceid|guid|firm|diag|dimm|bios|mparom|mac|all}
```

**Comment by tingtli** [comment]
Seems diag and mparom are also not supported.

**Comment by tingtli** [comment]
@gurevichmark  Would you also please take a look at the option explanation. Seems we don't have dimm and mparom explanation in the manpage details.

**Comment by gurevichmark** [comment]
@tingtli Do we have any systems that are "BMC/MPA" managed ?
@zet809 Which xCAT .pm module handles the processing of "BMC/MPA" managed computes ?

**Comment by tingtli** [comment]
You can take my nodes. MN:c910f04x36. cn:c910f04x29

**Comment by gurevichmark** [comment]
@tingtli and @zet809 I looked at the node definition for c910f04x29. It has `mgt=ipmi`.
Does this mean MPA is a x86 architecture BMC, managed using ipmi ?

**Comment by tingtli** [comment]
@gurevichmark  MPA is for Flex system.  Seems we don't have flex system now. c910f04x29 is a x86 architecture server which was managed by BMC with ipmi protocols.

**Comment by whowutwut** [comment]
We do. @neo954 has some. Frame5 u1 is a flex chassis

**Comment by gurevichmark** [comment]
Fixed by PR #3307

**Comment by tingtli** [comment]
I've verified with the latest build. It's ok to close
```
[root@c910f04x36 2.12]# lsxcatd -v
Version 2.13.5 (git commit 6ed46ff77f32042ccb45bc61c058e10f51a6678c, built Thu Jun 22 06:15:48 EDT 2017)

[root@c910f04x36 2.12]# man rinv
   BMC/MPA specific:
       rinv noderange [model|serial|asset|vpd|deviceid|guid|firm|dimm|mprom|all]

       mprom  Retrieves mprom firmware level.

       dimm   Retrieves dual in-line memory module information.

[root@c910f04x36 2.12]# rinv -h
    BMC/MPA specific:
       rinv <noderange> [model|serial|asset|vpd|deviceid|guid|firm|dimm|mprom|all]

```
